src/Containers/AddMovie/AddMovie.module.css

```diff
@@ -0,0 +1,11 @@
+.headerButton {
+  position: absolute;
+  top: 0;
+  right: 0;
+  margin-top: 10px;
+  margin-right: 10px;
```

Comment:
Positioning styles like this should come from outside so that a component can be more reusable.